pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -156,8 +156,20 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			},
 			Rules: []authorizationapi.PolicyRule{
 				{
-					Verbs:     util.NewStringSet(authorizationapi.VerbAll),
-					Resources: util.NewStringSet(authorizationapi.ResourceAll),
+					// replicationControllerGetter
+					Verbs:     util.NewStringSet("get", "list"),
+					Resources: util.NewStringSet("replicationcontrollers"),
+				},
+				{
+					// RecreateDeploymentStrategy.replicationControllerClient
+					// RollingDeploymentStrategy.updaterClient
+					Verbs:     util.NewStringSet("get", "update"),
+					Resources: util.NewStringSet("replicationcontrollers"),
+				},
+				{
+					// RecreateDeploymentStrategy.hookExecutor
+					Verbs:     util.NewStringSet("get", "create"),
```

Comment:
No "list"?  I was pretty sure I reviewed some deploy code that was doing a list of pods in an error case.